platform_channels/lib/src/pet_list_screen.dart

```diff
@@ -52,7 +53,7 @@ class _PetListScreenState extends State<PetListScreen> {
       floatingActionButton: FloatingActionButton(
         child: const Icon(Icons.add),
         onPressed: () {
-          Navigator.pushNamed(context, '/addPetDetails');
+          context.push('/addPetDetails');
```

Comment:
Note that here I use `push` instead of `go` because the route `/addPetDetails` is not part of `/petListScreen`, so to support navigating back, push is used instead.

Reply:
I changed this now, so it uses `go` as well.